### ISSUE TITLE
CLI breaks with non-ASCII characters

### ISSUE THREAD

**Issue by garberg** [SpriteLink/NIPAP#974]
A new problem with non-ASCII characters in the CLI has been introduced:

```
$ nipap a a prefix 10.0.0.0/24 type reservation description åäö
Network 10.0.0.0/24 added to VRF 'default' [RT: -]: åäö

$ nipap a l 10.0.0.0/24
Searching for prefixes in any VRF...
Traceback (most recent call last):
  File "/usr/bin/nipap", line 131, in <module>
    cmd.exe(cmd.arg, cmd.exe_options, args)
  File "/usr/lib/python2.7/dist-packages/nipap_cli/nipap_cli.py", line 489, in list_prefix
    len(str(val)))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-2: ordinal not in range(128)
```

**Comment by garberg** [comment]
OK, added a pull request for this. Two additional considerations:
* As this isn't the first time we're breaking non-ASCII characters we should probably look into a way to test this. Perhaps BATS (https://github.com/sstephenson/bats - available in main Debian & Ubuntu repos) might be useful?
* Perhaps we should consider letting the NIPAP CLI be Python 3 only to get rid of this shite once and for all.

**Comment by plajjan** [comment]
Fixed by #976. Closing.
